formatting-stack.edn

```diff
@@ -0,0 +1,3 @@
+{:files                                 ["src/" "test/"]
+ :formatting-stack.linters.kondo/config #include ".clj-kondo/config.edn"
```

Comment:
reading the config from `.clj-kondo/config.edn` is an example of the flexibility of this approach; it allows us to tap into the existing configuration. So devs can use `clj-kondo` from the CLI in this project too; with identical config